src/main/java/lavalink/client/io/Link.java

```diff
@@ -63,6 +63,13 @@ public Lavalink getLavalink() {
         return lavalink;
     }
 
+    public LavalinkRestClient getRestClient() {
```

Comment:
This method is problematic as it is prone to leaky abstractions.

If I was a user, I would expect this to work even if I replace all my websockets.

Reply:
How will that break? I'm slightly confused

Reply:
Ah my bad, I misread the code.

Reply:
It's all cool